its/tests/src/test/java/com/sonar/orchestrator/OrchestratorBuilder.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * SonarLint Core - ITs - Tests
+ * Copyright (C) 2016-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package com.sonar.orchestrator;
+
+import com.sonar.orchestrator.config.Configuration;
+import com.sonar.orchestrator.container.Edition;
+import com.sonar.orchestrator.container.SonarDistribution;
+import com.sonar.orchestrator.locator.FileLocation;
+import com.sonar.orchestrator.locator.Location;
+import com.sonar.orchestrator.server.StartupLogWatcher;
+import com.sonar.orchestrator.util.OrchestratorUtils;
+import com.sonar.orchestrator.util.System2;
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import javax.annotation.Nullable;
+
+public class OrchestratorBuilder {
```

Comment:
@damien-urruty-sonarsource I am in the process of fixing Orchestrator, so this class can be removed later.